src/mnode/mnode-pastel.h

```diff
@@ -618,3 +619,1814 @@ class CTakeDownTicket : public CPastelTicket
 
     void SerializationOp(CDataStream& s, const SERIALIZE_ACTION ser_action) override {}
 };
+
+// Username Change Ticket /////////////////////////////////////////////////////////////////////////////////////////////////////
+/*
+	"ticket": {
+		"type": "username",
+		"pastelID": "",    //PastelID of the username
+		"username": "",    //new valid username
+		"fee": "",         // fee to change username
+		"signature": ""
+	},
+ */
+class CChangeUsernameTicket : public CPastelTicket
+{
+public:
+    std::string pastelID;
+    std::string username;
+    CAmount fee{100};
+    std::vector<unsigned char> signature;
+
+public:
+    CChangeUsernameTicket() = default;
+
+	explicit CChangeUsernameTicket(std::string _pastelID) :
+        pastelID(std::move(_pastelID))
+    {}
+    std::string KeyOne() const noexcept override { return username; }
+    std::string KeyTwo() const noexcept override { return pastelID; }
+
+    bool HasKeyTwo() const noexcept override { return true; }
+    bool HasMVKeyOne() const noexcept override { return false; }
+    bool HasMVKeyTwo() const noexcept override { return false; }
```

Comment:
It is needed as a good practice.
override keyword served two purposes:

It shows the reader of the code that "this is a virtual method, that is overriding a virtual method of the base class."
The compiler also knows that it's an override, so it can "check" that you are not altering/adding new methods that you think are overrides.
So, place it here makes the code easier to read.
We can use "final" instead of "override". Yes. But it might leads to another unwanted things. For example, some unit tests want to mock this class to test, and by defining the function as final, we can't inherit the function again to do that.
For conclusion, using "override" is the best way I believe.